Index: blktap-2.0.90/configure.ac
===================================================================
--- blktap-2.0.90.orig/configure.ac	2011-11-28 17:56:57.000000000 +0000
+++ blktap-2.0.90/configure.ac	2012-10-11 16:27:32.279320907 +0000
@@ -52,7 +52,7 @@
 AM_CONDITIONAL([ENABLE_VHDIO],
 	       [test x$enable_shared = xyes &&
 	       	case "${host_os}" in
-		      linux-gnu) true ;;
+		      linux-gnu*) true ;;
 		      *) false ;;
 		esac])
 
